app/assets/javascripts/application.js

```diff
@@ -2,6 +2,7 @@
 //= require turbolinks
 //= require i18n
 //= require jquery.ui.autocomplete
+//= require jquery.ui.spinner
```

Comment:
Copying answer from first PR https://github.com/theforeman/foreman/pull/3774#discussion_r77506994

> patternfly spinner didn't allow me to customize the output and parsing - note that gb/mb is part of the input so user can type "5gb" and it will get parsed correctly

This cpu spinner just reuses some code that was added for the more advanced spinner. If there's a way to use patternfly one for this I'm happy to refactor this in next Foreman version. That should not block this PR I hope.

Here's the gif for simple CPU fields, changing the values using up/down keys

![vmcpu](https://cloud.githubusercontent.com/assets/109773/18271686/b8a49f0a-7433-11e6-8873-92557c7250c7.gif)

Here's the RAM being added in the first PR 
![vmram](https://cloud.githubusercontent.com/assets/109773/18271405/30569e10-7432-11e6-8913-25add74855f2.gif)